HealthMonitoring.UI.UnitTests/Spec/advanced-directives.spec.js

```diff
@@ -0,0 +1,100 @@
+﻿///<reference path="~/../HealthMonitoring.SelfHost/Content/Scripts/angular.min.js"/>
+///<reference path="~/Lib/angular-mocks.js"/>
+///<reference path="~/../HealthMonitoring.SelfHost/Content/Scripts/advanced-directives.js"/>
+
+describe("wgMultiselect directive test", function() {
+    var rootScope,
+        compile,
+        directive,
+        scope;
+
+    function getDirectiveHtml() {
```

Comment:
I added html directly into code because I could not load it via httpGet request. 
However, I don't like this hard-coded html too, so I could rewrite this test on selenium.